backend/src/test/java/heartbeat/service/source/github/GithubServiceTest.java

```diff
@@ -274,9 +275,51 @@ void CommitTimeInPrShouldBeZeroWhenCommitInfoIsNull() {
 		assertEquals(expect, result);
 	}
 
+	@Test
+	void CommitTimeInPrLeadTimeShouldBeZeroWhenCommitInfoIsRevert() {
```

Comment:
@PengxiWPix 
update test description to `should ....when...given`, not start a customized message